app/features/project/project_controller.rb

```diff
@@ -43,7 +48,8 @@ class ProjectController < AuthenticatedControllerBase
         project: project,
         sha: current_sha,
         github_service: FastlaneCI::GitHubService.new(provider_credential: current_github_provider_credential),
```

Comment:
new line here I think?

Reply:
Ahh, it's not over 120 characters, so I didn't break it up. Would you like me to?